pkgs/top-level/ocaml-packages.nix

```
@@ -751,7 +751,9 @@ let
 
     # Jane Street
     janePackage =
-      if lib.versionOlder "4.13.1" ocaml.version
+      if lib.versionOlder "5.0" ocaml.version
```

```suggestion
      if lib.versionOlder "5.1" ocaml.version
```

To be consistent with the minimum set in `janePackage_0_17.nix`.

---

Will this work for `ocaml.version` that is exactly `5.1`?

```
nix-repl> nixpkgs.lib.versionOlder "5.1" "5.1"
false
```

---

You might be right. Can `ocaml.version` be `"5.1"`?